ui-components/src/global/markdown.scss

```diff
@@ -1,53 +1,173 @@
 .markdown {
+  /* Set sizing and gaps for built-in HTML elements */
+
   h2 {
-    @apply mt-8;
-    @apply mb-5;
-    @apply text-3xl;
+    margin-block: var(--bloom-s8) var(--bloom-s5);
+    font-size: var(--bloom-font-size-3xl);
 
-    @screen md {
-      @apply text-xl;
+    @media (min-width: $screen-md) {
+      font-size: var(--bloom-font-size-xl);
     }
   }
 
   h2:first-child {
-    @apply mt-0;
+    margin-block-start: 0;
   }
 
   hr {
-    @apply mt-8;
-    @apply mb-8;
-    @apply border-b;
-    @apply border-solid;
-    @apply border-gray-450;
+    margin-block: var(--bloom-s8);
+    border-block-end: 1px solid var(--bloom-color-gray-450);
   }
 
   p:not(:last-child),
   ul:not(:last-child) {
-    @apply mb-4;
+    margin-block-end: var(--bloom-s4);
   }
 
   ul {
     list-style: disc;
-    @apply ml-6;
+    margin-inline-start: var(--bloom-s6);
   }
 
+  /* Certain text blocks in the application form flow use this extra styling: */
+
   &.markdown-informational {
-    font-size: 0.9em;
-    @apply text-gray-750;
+    font-size: var(--bloom-font-size-tiny);
+    color: var(--bloom-color-gray-750);
 
     h3 {
-      @apply font-sans;
-      @apply font-bold;
-      @apply text-base;
-      @apply text-black;
+      font-family: var(--bloom-font-sans);
+      font-weight: bold;
+      font-size: var(--bloom-font-size-base);
+      color: var(--bloom-color-black);
+      margin-block-end: var(--bloom-s4);
     }
 
     li {
-      @apply mb-4;
+      margin-block-end: var(--bloom-s4);
     }
 
     a {
-      @apply underline;
+      text-align: underline;
+    }
+  }
+
+  /* Used for a larger lead paragraph: */
```

Comment:
In the future, let's separate out changes like this into a different PR so that we can make an issue, discuss it in planning, point it, shorten the review scope, and make sure that the point estimate we made for the initial issue in planning stays accurate :)

Reply:
Oh, I forgot to mention https://github.com/housingbayarea/bloom/issues/492 in the PR description. It was something that already come up with a previous review round…I didn't want to just port the Markdown styles over from HBA without at least modernizing it a bit, and I do think we should create another issue for further review of our Markdown styles (part of the audit?)